src/main/java/picard/sam/SamToFastq.java

```diff
@@ -245,11 +215,8 @@ protected int doWork() {
         return 0;
     }
 
-    /**
-     * Generates the writers for the given read groups or, if we are not emitting per-read-group, just returns the single set of writers.
-     */
-    private Map<SAMReadGroupRecord, FastqWriters> generateWriters(final List<SAMReadGroupRecord> samReadGroupRecords,
-                                                                  final FastqWriterFactory factory) {
+    private Map<SAMReadGroupRecord, FastqWriters> generateWriters(List<SAMReadGroupRecord> samReadGroupRecords,
+                                                                  FastqWriterFactory factory) {

```

Comment:
`factory` is no longer final? Also why remove javadoc?

Reply:
hmm I have no idea how i managed to do that.....